src/components/Header/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Dropdown } from '@edx/paragon';
+
+import Img from '../Img';
+import EdxLogo from '../../images/edx-logo.png';
+import './Header.scss';
+
+const Header = (props) => {
+  const {
+    enterpriseLogo,
+    enterpriseSlug,
+    email,
+    logout,
+  } = props;
+
+  return (
+    <header className="container">
+      <nav className="navbar px-0 justify-content-between">
+        <Link
+          to={enterpriseSlug ? `/${enterpriseSlug}` : '/'}
+          className="navbar-brand"
+        >
+          <Img src={enterpriseLogo || EdxLogo} alt="" />
+        </Link>
+        {email && <Dropdown
+          title={email}
+          menuItems={[
+            <Link to="/login" onClick={logout}>Logout</Link>,
```

Comment:
I kept this as a link to `/login` but with the onClick removing the cookie.  Other alternatives we could consider:

- Redirect in the action handler for logout
- Have a /logout route that removes the cookie and redirects

I was a worried with the currently implementation we'd get to the login page and still have isAuthenticated to true and bypass the page, before the action handler resolved. It seems to work in testing it, but it still feels a bit messy.

Reply:
yea, I would have expected a /logout route - that would make it easier to reuse the functionality if we need to trigger logout in other situations.